server/src/models/user.model.ts

```diff
@@ -0,0 +1,85 @@
+import {Entity, model, property, belongsTo} from '@loopback/repository';
+import {Customer} from './customer.model';
+import {Role} from './role.model';
+
+@model({settings: {strict: true}})
```

Comment:
provide name parameter here
and other modesl also

Reply:
provided the name of each model